src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  --button-color: #fff;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
```

Comment:
1) your card should have all sides paddings. 

2) in this case children only need to have an indent with margin-bottom
Example:

```
.card {
  padding: 20px
}

.card__img {
  margin: 0 auto 40px;
}

.card__title {
  margin-bottom: 40px;
}

.card__subtitle {
  margin-bottom: 20px;
}
```